File: PodcastChapters/Utilities/Extensions/SBApplication+PCH.m
//
//  SBApplication+PCH.m
//  PodcastChapters
//
//  Created by Szabolcs Toth on 2016. 10. 03..
//  Copyright © 2016. Szabolcs Toth. All rights reserved.
//

#import "iTunes.h"
#import "SBApplication+PCH.h"

@implementation SBApplication (PCH)

+ (iTunesApplicationWrapper *)pch_iTunes
{
    iTunesApplication *app = (iTunesApplication *)[SBApplication applicationWithBundleIdentifier:iTunesBundleIdentifier];
    return [[iTunesApplicationWrapper alloc] initWithItunesApplication:app];
}

@end

File: PodcastChapters/Utilities/Extensions/SBApplication+PCH.swift
//
//  SBApplication+PCH.swift
//  PodcastChapters
//
//  Created by Szabolcs Toth on 2016. 02. 09..
//  Copyright © 2016. Szabolcs Toth. All rights reserved.
//

import ScriptingBridge

extension SBApplication {

    class func pch_iTunes() -> iTunesApplication {
        return SBApplication(bundleIdentifier: iTunesApp.BundleIdentifier) as! iTunesApplication
    }
}
